src/utilities/apiCalls.js

```diff
@@ -0,0 +1,26 @@
+const getFriends = () => {
+  return fetch("https://gifty-backend-rails.herokuapp.com/api/v1/users/1/friends")
+    .then(response => response.json())
+};
+
+const postFriend = (friend) => {
+  return fetch('https://gifty-backend-rails.herokuapp.com/api/v1/users/1/friends', {
+    method: 'POST',
+    body: JSON.stringify({
+      name: friend.name,
+      birthday: friend.birthday,
+      memo: friend.memo,
+      need_gift: friend.need_gift
+    }),
+    headers: {
+      "Content-Type": "application/json"
+    }
+  })
+    .then(response => response.json())
+};
+
+
```

Comment:
We should add a helper function for error handling on these, and also a `.catch`. Might be a good idea to open an error handling branch and we can do the error handling for all of the forms/network requests at the same time.